ui/app/mirrors/edit/[mirrorId]/cdcDetails.tsx

```diff
@@ -1,35 +1,76 @@
+'use client'
+import { Label } from '@/lib/Label';
+import { Badge } from '@/lib/Badge';
+import { Icon } from '@/lib/Icon';
+import {Action} from '@/lib/Action';
 import { FlowConnectionConfigs } from '@/grpc_generated/flow';
+import CdcGraph from './cdcGraph'
+import moment from 'moment';
 
-type CDCDetailsProps = {
-  config: FlowConnectionConfigs | undefined;
+
+type SyncStatusRow = {
+  batchId: number;
+  startTime: Date;
+  endTime: Date | null;
+  numRows: number;
 };
 
-export default function CDCDetails({ config }: CDCDetailsProps) {
-  if (!config) {
-    return <div className='text-red-500'>No configuration provided</div>;
-  }
+type props = {
+    syncs: SyncStatusRow[];
+    mirrorConfig: FlowConnectionConfigs | undefined;
+  };
+function CdcDetails({ syncs,mirrorConfig }:props) {
+
+  let lastSyncedAt = moment(syncs[0]?.startTime).fromNow();
+  let rowsSynced = syncs.reduce((acc, sync) => acc + sync.numRows, 0)
 
   return (
-    <div className='p-4 rounded-md'>
-      <h2 className='text-xl font-semibold mb-4'>CDC Details</h2>
-      <div className='overflow-x-auto'>
-        <table className='min-w-full divide-y divide-gray-300'>
-          <tbody>
-            <tr>
-              <td className='px-4 py-2 font-medium'>Source</td>
-              <td className='px-4 py-2'>{config.source?.name || '-'}</td>
-            </tr>
-            <tr>
-              <td className='px-4 py-2 font-medium'>Destination</td>
-              <td className='px-4 py-2'>{config.destination?.name || '-'}</td>
-            </tr>
-            <tr>
-              <td className='px-4 py-2 font-medium'>Flow Job Name</td>
-              <td className='px-4 py-2'>{config.flowJobName}</td>
-            </tr>
-          </tbody>
-        </table>
+    <>
+    <div className='mt-10'>
+      <div className="flex flex-row">
+        <div className="basis-1/4 md:basis-1/3">
+          <div><Label variant="subheadline" colorName='lowContrast'>Status</Label></div>
+          <div><Label variant="body">            
+            <Badge variant='positive' key={1}>
+              <Icon name='play_circle' />
+              Active
+            </Badge>
+            </Label>
+          </div>
+        </div>
+        <div className="basis-1/4 md:basis-1/3">
+          <div><Label variant="subheadline" colorName='lowContrast'>Mirror Type</Label></div>
+          <div><Label variant="body">CDC</Label></div>
+        </div>
+        <div className="basis-1/4 md:basis-1/3">
+          <div><Label variant="subheadline" colorName='lowContrast'>Source</Label></div>
+          <div><Action href={"/peers/"+mirrorConfig?.source?.name}>{mirrorConfig?.source?.name}</Action></div>
```

Comment:
Here and for the destination we can use `<PeerButton peerName={mirrorConfig?.source?.name} peerType={mirrorConfig?.source?.type}/>` (located in `/components/PeerComponent.tsx`